spark/src/test/scala/org/apache/spark/sql/delta/stats/StatsCollectionSuite.scala

```diff
@@ -499,14 +499,13 @@ class StatsCollectionSuite
     ("BINARY", "BinaryType"),
     ("BOOLEAN", "BooleanType"),
     ("ARRAY<TINYINT>", "ArrayType(ByteType,true)"),
-    ("MAP<DATE, INT>", "MapType(DateType,IntegerType,true)"),
-    ("STRUCT<c60:INT, c61:ARRAY<INT>>", "ArrayType(IntegerType,true)")
+    ("MAP<DATE, INT>", "MapType(DateType,IntegerType,true)")
```

Comment:
Could we also look at other tests in StatsCollectionSuite and update them, if applicable, for our use-cases?

Examples:
https://github.com/delta-io/delta/blob/da5a5d29b6f00265439bc84020580080a2d4568f/spark/src/test/scala/org/apache/spark/sql/delta/stats/StatsCollectionSuite.scala#L611
https://github.com/delta-io/delta/blob/da5a5d29b6f00265439bc84020580080a2d4568f/spark/src/test/scala/org/apache/spark/sql/delta/stats/StatsCollectionSuite.scala#L611

Reply:
Added more valid types to the test